src/main/java/com/google/sps/data/SmallCityService.java

```diff
@@ -63,14 +58,10 @@ public void eliminateBigBusinesses() {
     return businesses;
   }
 
-  private void addListingToBusinesses(PlacesSearchResult place) {
-    String name = place.name;
-    String formattedAddress = place.vicinity;
-    Geometry geometry = place.geometry;
-    MapLocation placeLocation = new MapLocation(geometry.location.lat, geometry.location.lng);
-    double rating = place.rating;
-    Photo photos[] = place.photos;
-    String types[] = place.types;
-    businesses.add(new Listing(name, formattedAddress, placeLocation, rating, photos, types));
+  private void createBusinessesService() {
```

Comment:
Rename this to focus on the goal from the application's pov, not the implementation. Something like "getSmallBusinesses()" etc..

Reply:
Changed! Thank you!